source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+[[contributing-user-stories]]
+= Contribute user stories to oVirt documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for oVirt GitHub repository
+
```

Comment:
Do you need context information here?

Reply:
Honestly, I'm not familiar with this attribute.

Reply:
I don't see a reason for it here.